lib/routers/establishment/projects.js

```diff
@@ -23,6 +23,10 @@ const submit = action => (req, res, next) => {
         case 'create':
           return req.workflow.create(params);
         case 'delete':
+          if (req.project.status !== 'inactive') {
+            throw new Error('Active projects cannot be deleted.');
```

Comment:
Can this throw a `BadRequestError` rather than just a normal error please? That way it will return a 400 error rather than a 500. Thanks.

Reply:
what about expired/revoked? Shouldn't we _just_ check project isn't active?

Reply:
No, this is correct. Expired and revoked projects shouldn't be delete-able.